language/vm/src/normalized.rs

```diff
@@ -108,6 +108,20 @@ impl Module {
             public_functions,
         }
     }
+
+    /// Return the private functions of module `m`
```

Comment:
I don't see this new function being used anywhere. Same for the is_copy function below. Should those be part of a separate PR?

Reply:
My bad; I neglected to check in the relevant file.